### ISSUE TITLE
3.7.3 for private mirrors?

### ISSUE THREAD

**Issue by tonyfinn** [eslint/eslint-scope#42]
From this comment: https://github.com/eslint/eslint-scope/issues/39#issuecomment-404533026

> Now that 3.7.2 is unpublished and 4.0.0 is latest, I'm not sure if we need to publish a 3.7.3, but please let us know if you think we might.

After the compromised version of 3.7.2 was released, the package was removed from the public npm registry, which is good. But many companies run private npm mirrors, and after the `left-pad` incident, many of these will not mirror deletes from the public npm repo.

As such, while the original decision was leaning towards not publishing a 3.7.3, I think that a 3.7.3 should be published as not all these teams will have pinned versions, and may only be pulling in eslint-scope via eslint or webpack, thereby getting the compromised version from their own mirrors.

(Posted as a new issue because the original thread was locked)

**Comment by platinumazure** [comment]
Hi @tonyfinn, thanks for the issue. We're working on this now. I'll update the original issue to note that we're working on publishing new patch versions.

**Comment by obviouslyworking** [comment]
Our mirror has picked up a 3.7.3
@platinumazure is the 3.7.3 package safe?

**Comment by not-an-aardvark** [comment]
Yes, see https://github.com/eslint/eslint-scope/issues/39#issuecomment-404597445.
